bot/utils/audio_recognition.py

```diff
@@ -0,0 +1,72 @@
+import os
+from google.cloud import speech
+import requests
+import logging
+import json
+import io
+from pydub import AudioSegment
+
+TOKEN = os.environ["TOKEN"]
+LANG = os.environ["LANG"]
+SAMPLE_RATE_HERTZ = os.environ["SAMPLE_RATE_HERTZ"]
```

Comment:
It is ok to fix these settings as a constants right here. No need to put them in env.

Reply:
Hi! No problem, I'll fix it this evening. How about converting from .oga to .wav? Any ideas?

Reply:
ffmpeg filexxx.oga /tmp/blabla/filexxx.wav

Reply:
We will need to add ffmpeg to docker image and mount some temp directory from host.